playback/src/player.rs

```diff
@@ -1329,11 +1335,8 @@ impl PlayerInternal {
                                     // step 4: subtractor
                                     let limiter_input = abs_sample_db - limited_sample;
 
-                                    // Spare the CPU unless the limiter is active or we are riding a peak.
-                                    if !(limiter_input > 0.0
-                                        || self.normalisation_integrator > 0.0
-                                        || self.normalisation_peak > 0.0)
-                                    {
+                                    // Spare the CPU unless the limiter is active.
+                                    if limiter_input <= 0.0 {
```

Comment:
This is incorrect because `limiter_input` is always `>= 0.0`.

Even when `limiter_input` is `0.0` (meaning that the current sample is at or below threshold) you may want to continue, because we may still have to release a peak we were just in.

Reply:
I was just going by your code. `limiter_input <= 0.0` is the equivalent of `!(limiter_input > 0.0)` without the `!`.

Reply:
True, _but_ I think it misconveys the intention. Comparing against `> 0.0` is a shorthand for non-zero without the epsilon cruft. Comparing against `<= 0.0` is the same logically but can give the impression that the value could be negative, while we know it can't.